src/lib/drivers/driver.js

```diff
@@ -143,8 +143,11 @@ class DriverBase {
         includeCommandLineAPI: true
       });
 
-      // If this gets to 15s and it hasn't been resolved, reject the Promise.
-      asyncTimeout = setTimeout(reject, 15000);
+      // If this gets to 60s and it hasn't been resolved, reject the Promise.
+      asyncTimeout = setTimeout(
+        _ => reject(new Error('aXe did not complete in the allotted time of 60s')),
```

Comment:
> It may be worth looking into turning off aXe for certain sites, or piping a way to configure this timeout on a per test basis.

@brendankenny is working on test whitelisting and will be putting a proposal for how that works soon.